radixdlt/src/integration/java/com/radixdlt/consensus/deterministic/ControlledNetwork.java

```diff
@@ -170,63 +196,66 @@ public String toString() {
 		}
 	}
 
-	public ControlledSender getSender(ECPublicKey sender) {
+	public ControlledSender createSender(ECPublicKey sender) {
 		return new ControlledSender(sender);
 	}
 
 	public final class ControlledSender implements BFTEventSender, VertexStoreEventSender, SyncVerticesRPCSender, EpochChangeSender,
-		CommittedStateSyncSender {
+		CommittedStateSyncSender, LocalTimeoutSender {
 		private final ECPublicKey sender;
+		private EpochAndView currentEpochAndView = new EpochAndView(1L, View.genesis());
```

Comment:
Perhaps this can be implemented better down the line by using the internal messages to synchronize the epochAndView. An adversary should have read only access to any node anyway.

Reply:
Not sure that it matters here, as this is only used for message ordering in any case, and isn't exposed outside of the package.  Might rename the class so that it's clear that it is for message ordering.

Reply:
Renamed the class in #143